### PR TITLE
ENH make sure cos7 stuff works

### LINKED ISSUE #1356: fix up cos7 conda_build_config files
Right now we see a bunch of errors due to missing keys in the conda_build_config.yaml files for cos7. Mostly this is cudnn.

We should write a minimigrator to fix this so that they all look like this:

```
cudnn:                                    # [linux64]
  - undefined                             # [linux64]
cuda_compiler_version:                    # [linux64]
  - None                                  # [linux64]
docker_image:                             # [linux64]
  - condaforge/linux-anvil-cos7-x86_64    # [linux64]
cdt_name:                                 # [linux64]
  - cos7                                  # [linux64]
```

cc @jakirkham

### PR DESCRIPTION
**Pull request by beckermr**
This PR makes sure that feedstocks with cos7 have the right config so they can be rerendered properly.

closes #1356

### CODE REVIEW

**Comment by beckermr** [comment]
@CJ-Wright this one is ready for review!

**Comment by beckermr** [comment]
ok - i'm gonna ship this. If we see issues I will take a look.

File: conda_forge_tick/migrators/cos7.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        if any(
+            SYSROOT_REGEX.search(line) for line in attrs["raw_meta_yaml"].splitlines()
+        ):
+            return False
+        else:
+            return True
+
+    def migrate(self, recipe_dir: str, attrs: "AttrsTypedDict", **kwargs: Any) -> None:
+        with indir(recipe_dir):
+            cfg = "conda_build_config.yaml"
```
Comment (isuruf) [inline conda_forge_tick/migrators/cos7.py:68]: This file should be checked for existence given https://github.com/conda-forge/conda-forge.github.io/pull/1264

Comment (beckermr) [reply conda_forge_tick/migrators/cos7.py:68]: See https://github.com/regro/cf-scripts/pull/1366

Comment (isuruf) [reply conda_forge_tick/migrators/cos7.py:68]: Thanks

**Pull request merged**